dlk/python/dlk/templates/include/de10_nano.h

```diff
@@ -505,28 +505,22 @@ Parameters calcParameters(uint32_t inputHeight, uint32_t inputWidth, uint32_t in
 }
 
 void RunTCA(unsigned long input_addr, unsigned long output_addr, unsigned long kernel_addr,
-  BIN_CONV_OUTPUT th_data[], unsigned in_w, unsigned in_h, unsigned in_c, unsigned nbits_in_data,
+  unsigned long thresholds_addr, unsigned in_w, unsigned in_h, unsigned in_c, unsigned nbits_in_data,
   unsigned out_w, unsigned out_h, unsigned out_c, unsigned k_w, unsigned k_h, unsigned pad, unsigned stride) {
 
   const unsigned k_size = (k_h * k_w * in_c * out_c) / 32;
   // MappedMem k_data_mem(KERNEL_ADDR, k_size, sizeof(T_UINT));
   // k_data_mem.Write(k_data_packed, k_size);
```

Comment:
These commented lines should be kept, or better to remove?

Reply:
Removed.